packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -257,7 +257,7 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
   const isInLocalHold = currentPage === 'hold';
   const hasJoinedCall = !!(currentPage && hasJoinedCallFn(currentPage, currentCallState ?? 'None'));
-  const showControlBar = isInLobbyOrConnecting || hasJoinedCall;
+  const showControlBar = isInLobbyOrConnecting || hasJoinedCall || currentCallState === 'Disconnecting';
```

Comment:
Should we consider "disconnecting" still a part hasJoinedCall state in line 259?

Reply:
Yep I like that better, updated